src/admin-dhis2/admin-dhis2.routes.js

```diff
@@ -26,7 +26,7 @@
         $stateProvider.state('openlmis.administration.dhis2', {
             url: '/dhis2',
             label: 'admin.dhis2.label',
-            isOffline: false,
+            isOffline: true,
```

Comment:
this should be false, we don't support offline mode for DHIS2

Reply:
Sure, thanks for the information about that, I have set up as true and thought this one needs to be true, will update!